http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -1229,14 +1238,18 @@ class GKEInterpreter[F[_]](
     )
   }
 
-  private[util] def buildCromwellAppChartOverrideValuesString(appName: AppName,
-                                                              cluster: KubernetesCluster,
-                                                              nodepoolName: NodepoolName,
-                                                              namespaceName: NamespaceName,
-                                                              disk: PersistentDisk): List[String] = {
+  private[util] def buildCromwellAppChartOverrideValuesString(
+    appName: AppName,
+    cluster: KubernetesCluster,
+    nodepoolName: NodepoolName,
+    namespaceName: NamespaceName,
+    disk: PersistentDisk,
+    customEnvironmentVariables: Map[String, String]
+  ): List[String] = {
     val proxyPath = s"/proxy/google/v1/apps/${cluster.googleProject.value}/${appName.value}/cromwell-service"
     val k8sProxyHost = kubernetesProxyHost(cluster, config.proxyConfig.proxyDomain).address
     val leoProxyhost = config.proxyConfig.getProxyServerHostName
+    val gcsBucket = customEnvironmentVariables.getOrElse("WORKSPACE_BUCKET", "")
```

Comment:
Feels like a slightly weird way of accessing the bucket (via environment variables that might be passed to the pod, but aren't, and we fish a specific value out of them instead and throw the rest away)...

I'm not saying it's bad, just surprising. Can we look at the code that puts the bucket _into_ the environment variables list in the first place and look up the bucket in the same way?

Reply:
A missing bucket value should probably be a failure to start the instance rather than passing along an empty value and letting Cromwell fail later on

Reply:
So are you asking that we find the customEnvironmentVariables function in the code and see if there is an alternative?

Reply:
That's kind of where I was going... it wouldn't be the end of the world to merge as is, so don't spend too long on it, this way just seemed a little odd to me

Reply:
I see. The function `buildGalaxyChartOverrideValuesString` did a similar thing for the workspace name and workspace namespace in the same file (line 1293 & 1294), but it would be interesting to see this done in the way you suggested.